src/page-modules/assistant/trip/trip-pattern/trip-pattern-header/index.tsx

```diff
@@ -15,7 +15,11 @@ type TripPatternHeaderProps = {
 export function TripPatternHeader({ tripPattern }: TripPatternHeaderProps) {
   const { t, language } = useTranslation();
 
-  const duration = secondsToDuration(tripPattern.duration, language);
```

Comment:
Should probably remove `duration` from the trip pattern object now if it's not in use

Reply:
Removed duration.